tests/Platforms/SQLitePlatformTest.php

```diff
@@ -304,9 +304,9 @@ public function testCreateTableWithDeferredForeignKeys(): void
                 . '"id" INTEGER NOT NULL, "article" INTEGER NOT NULL, "post" INTEGER NOT NULL'
                 . ', "parent" INTEGER NOT NULL, PRIMARY KEY("id")'
                 . ', CONSTRAINT "FK_8D93D64923A0E66" FOREIGN KEY ("article")'
-                . ' REFERENCES "article" ("id") DEFERRABLE INITIALLY IMMEDIATE'
+                . ' REFERENCES "article" ("id") DEFERRABLE'
                 . ', CONSTRAINT "FK_8D93D6495A8A6C8D" FOREIGN KEY ("post")'
-                . ' REFERENCES "post" ("id") NOT DEFERRABLE INITIALLY DEFERRED'
+                . ' REFERENCES "post" ("id") DEFERRABLE INITIALLY DEFERRED'
```

Comment:
`INITIALLY DEFERRED` implies `DEFERRABLE`. The previous implementation would render `NOT DEFERRABLE` (which SQLite tolerates) because the test doesn't explicitly set `['deferrable' => true]`. Now this is fixed.